homeassistant/components/huawei_lte/quality_scale.yaml

```diff
@@ -0,0 +1,76 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  entity-event-setup: done
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: todo
+  test-before-configure: done
+  test-before-setup: done
+  unique-config-entry: done
+
+  # Silver
+  action-exceptions: todo
+  config-entry-unloading: done
+  docs-configuration-parameters: done
+  docs-installation-parameters: done
+  entity-unavailable: done
+  integration-owner: done
+  log-when-unavailable:
+    status: todo
+    comment: Kind of done, but to be reviewed, there's probably room for improvement. Maybe address this when converting to use a data update coordinator. See also https://github.com/home-assistant/core/issues/55495
+  parallel-updates: todo
+  reauthentication-flow: done
+  test-coverage: todo
```

Comment:
Why does `test_clear_traffic_statistics` and `test_restart` both suspend the intergation at the end? what does that test?

Reply:
That code was written by someone else, and probably inadvertently passed review. I guess the intent was to suspend the integration and then invoke another button press, and assert that the button press does not yield its normal results. The button presses while suspended are missing from the test, though.

(Aside, I intend to remove the suspend/resume functionality sometime, IIRC there are better builtin ways to accomplish the same these days.)

Reply:
I was actually discussing the resume suspend functionality in the core team today, and we think that adding a service to enable/disable an entry via home assistant makes sense and would be a nice replacement for that service

Reply:
Oh, I thought something to that extent already existed. But cool.

Reply:
It is implemented in spook

Reply:
Hm, what does spook mean in this context?

Reply:
Anyway bug apparently missing presses noted in 6f4b51d1eebeb8405feeff141d01bde7000425b1